packages/theme-chalk/src/menu.scss

```diff
@@ -173,7 +173,8 @@
     );
 
     > .#{$namespace}-menu-item,
-    > .#{$namespace}-sub-menu > .#{$namespace}-sub-menu__title {
+    > .#{$namespace}-sub-menu > .#{$namespace}-sub-menu__title,
+    > .#{$namespace}-menu-item-group > ul > .#{$namespace}-sub-menu > .#{$namespace}-sub-menu__title {
```

Comment:
Is there a more elegant solution? It looks unstable. emmm

Reply:
The class name must be matched exactly here, otherwise an error may occur when configuring `:popper-append-to-body="false"`